docs/src/components/home/CoreShowcase.tsx

```diff
@@ -163,29 +165,10 @@ export default function CoreShowcase() {
               Custom Theme
             </Button>
           </Box>
-          <Box
-            sx={{
-              position: 'relative',
-              overflow: 'clip',
```

Comment:
We still need `overflow: clip` here, otherwise:

<img width="400" alt="Screenshot 2024-05-06 at 17 45 54" src="https://github.com/mui/material-ui/assets/67129314/9ca74796-5950-4895-889b-65b1e0226c70">

Reply:
I can't reproduce this.